src/app/case-studies/la-county/page.tsx

```diff
@@ -0,0 +1,143 @@
+import Link from 'next/link';
+import { Link as ExternalLink } from '@trussworks/react-uswds';
+import './styles.scss';
+
+export default function LaCountyCaseStudy() {
```

Comment:
The case study pages all have a very similar layout. When working on the next case study page, I'll start looking into breaking these sections out into layout components.